sample/pom.xml

```diff
@@ -24,7 +24,7 @@
             <groupId>com.microsoft.gctoolkit</groupId>
             <artifactId>gctoolkit-vertx</artifactId>
             <version>${project.version}</version>
-        </dependency>
+        </dependency><dependency><groupId>org.junit.jupiter</groupId><artifactId>junit-jupiter-api</artifactId></dependency><dependency><groupId>org.junit.jupiter</groupId><artifactId>junit-jupiter-engine</artifactId></dependency>
```

Comment:
Looks like whitespace got mangled here

Reply:
I'll undo that - sorry 'bout that

Reply:
IntelliJ added that dependency when I created the test package in the samples module.